resources/site-packages/quasar/navigation.py

```diff
@@ -134,12 +134,12 @@ def run(url_suffix=""):
         else:
             import traceback
             map(log.error, traceback.format_exc().split("\n"))
-            notify(getLocalizedLabel(unicode(e.reason, 'utf-8')), time=7000)
+            notify(e.reason, time=7000)
         return
     except Exception as e:
         import traceback
         map(log.error, traceback.format_exc().split("\n"))
-        notify(getLocalizedLabel(unicode(e, 'utf-8')), time=7000)
+        notify(e, time=7000)
```

Comment:
This needs to go through `getLocalizedLabel`, as I said I'm pretty sure this one is fine and shouldn't be changed.